Godeps/Godeps.json

```diff
@@ -1,7 +1,7 @@
 {
 	"ImportPath": "k8s.io/dns",
 	"GoVersion": "go1.8",
```

Comment:
Update to go1.9.3 to match what we have? Ref https://github.com/kubernetes/dns/pull/193.

Reply:
Actually, seems like we need to update the godep scripts https://github.com/kubernetes/dns/blob/d38b2c3779516e1e1d2748a6df0bd2e07937a5b9/build/dep.sh#L19.